plugin/wakatime.vim

```diff
@@ -144,17 +144,17 @@ let s:VERSION = '9.0.1'
     endfunction
 
     function! s:InstallCLI(use_external_python)
-        if !s:autoupdate_cli && filereadable(s:wakatime_cli)
+        if !s:autoupdate_cli && executable(s:wakatime_cli)
```

Comment:
We need to check `!empty(s:wakatime_cli)` before any `executable()` because of a Vim bug.

Reply:
Old bug? I don't know it.

Reply:
I am a rookie, can you tell me more?

Reply:
I try `:echom executable(1)` show `E928: String required` in nvim (v0.6.0-2.1.0-beta3).
vim (8.2.3850) no bug.

> In nvim, `executable()` param must be a string (include empty string).
> If set not string or empty string param, `empty()` will return 0.

Thank you fix it.